store/src/java/com/zimbra/cs/fb/ExchangeEWSFreeBusyProvider.java

```diff
@@ -920,42 +924,147 @@ public boolean handleMailboxChange(String accountId) {
             String fb = "";
             int i = 0;
             for (FreeBusyResponseType attendeeAvailability : results) {
-                if (re.email == attendees.getMailboxData().get(i).getEmail()
-                        .getAddress()) {
-                    if (ResponseClassType.SUCCESS != attendeeAvailability
-                            .getResponseMessage().getResponseClass()) {
-                        ZimbraLog.fb
-                                .warn(attendeeAvailability.getResponseMessage().getMessageText());
-                        ZimbraLog.fb
-                                .warn("Error in response. continuing to next one");
+            	String fbResponseViewType = attendeeAvailability.getFreeBusyView().getFreeBusyViewType().get(0);
+            	ZimbraLog.fb.info("fbResponseViewType : " + fbResponseViewType);
+            	
+                if (re.email == attendees.getMailboxData().get(i).getEmail().getAddress()) {
+                    if (ResponseClassType.SUCCESS != attendeeAvailability.getResponseMessage().getResponseClass()) {
+                        ZimbraLog.fb.warn(attendeeAvailability.getResponseMessage().getResponseCode()+"::"+attendeeAvailability.getResponseMessage().getMessageText());
```

Comment:
String format style, also consider making local variables out of this responseCode and messageText since it is used in a few places.
`ZimbraLog.fb.warn("%s::%s", attendeeAvailability.getResponseMessage().getResponseCode(), attendeeAvailability.getResponseMessage().getMessageText());`